src/strongbox/http.clj

```diff
@@ -9,12 +9,47 @@
    [me.raynes.fs :as fs]
    [orchestra.core :refer [defn-spec]]
    [trptcolin.versioneer.core :as versioneer]
-   [clj-http.client :as client]))
+   [clj-http
+    [core]
+    [util :refer [opt]]
+    [client :as client]])
+  (:import
+   [org.apache.http.client.config CookieSpecs RequestConfig]))
+
 
 ;; todo: revisit this value
 (def expiry-offset-hours 24) ;; hours
 (def ^:dynamic *cache* nil)
 
+;; todo: remove when upstream fix is in
+;; https://github.com/dakrone/clj-http/pull/583
+(defn request-config [{:keys [connection-timeout
+                              connection-request-timeout
+                              socket-timeout
+                              max-redirects
+                              cookie-spec
+                              ; deprecated
+                              conn-request-timeout
+                              conn-timeout]
+                       :as req}]
+  (let [config (-> (RequestConfig/custom)
+                   (.setNormalizeUri false)
```

Comment:
can this be tested?

Reply:
tried to, but something about clj-fakes means the redirect being returned isn't being followed.